### PR TITLE
[tests-only][full-ci] added test for mail notification if space admin delete access to space

### PR DESCRIPTION
**Pull request by ghost**
## Description
This PR adds the API test scenario for mail notification.
* Scenario: user gets an email notification when space admin unshares a space
-  [ ] forward port to master branch https://github.com/owncloud/ocis/pull/6719

## Related Issue
* Part of: https://github.com/owncloud/ocis/issues/6606

#### test environment:
* locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [X] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/bootstrap/SpacesContext.php [hunk 1]
```diff
@@ -204,6 +204,24 @@ public function getSpaceByName(string $user, string $spaceName): array {
 		return $spaces[$spaceName];
 	}
 
+	/**
+	 * The method finds available spaces to the user and returns the space by spaceId
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/SpacesContext.php:208]:
```suggestion
	 * The method finds the space of a user by name and returns the space id
```

File: tests/acceptance/features/bootstrap/SpacesContext.php [hunk 2]
```diff
@@ -204,6 +204,24 @@ public function getSpaceByName(string $user, string $spaceName): array {
...
+	 * @param string $spaceName
+	 *
+	 * @return string
+	 * @throws GuzzleException
+	 */
+	public function getSpaceIdByName(string $user, string $spaceName): string {
+		$space = $this->getSpaceByName($user, $spaceName);
+		Assert::assertIsArray($space, "Space with name $spaceName not found");
+		if (!isset($space["id"])) {
+			throw new Exception(__METHOD__ . " space with name $spaceName not found");
```
Comment (SagarGi) [inline tests/acceptance/features/bootstrap/SpacesContext.php:220]:
```suggestion
			throw new Exception(__METHOD__ . " space with name $spaceName not found in $space");
```

**Review by saw-jan — APPROVED** [review]
LGTM

File: tests/acceptance/features/apiNotification/emailNotification.feature [hunk 3]
```diff
@@ -157,3 +157,24 @@ Feature: Email notification
...
       """
+
+
+  Scenario: user gets an email notification when space admin unshares a space
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the GraphApi
+    And user "Alice" shares a space "new-space" with settings:
+      | shareWith | Brian  |
+      | role      | Editor |
+    And the HTTP status code should be "200"
```
Comment (SwikritiT) [inline tests/acceptance/features/apiNotification/emailNotification.feature:168]: Should his be a `Given` step?
```suggestion
   And user "Alice" has shared a space "new-space" with settings:
      | shareWith | Brian  |
      | role      | Editor |
```

**Review by SwikritiT — APPROVED** [review]
lgtm

**Pull request merged**